### PR TITLE
CHAT-1066: Fixing the following Css styles (24642)

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
34:The unread messages badge color in the Chat drawer is different than the same badge color in the Tpbar chat icon.

35: No hand seen at the 3 points of the options of the message to validate that those buttons are clickable.

36: When the name of the user is too long, it is not well displayed in the drawer ; must display three ellipsis,hence a bad arrangement of the close-open chat-filter buttons

37: The close Button of the drawer is smaller than the open chat & filter button.

[TRIBE-1363] App Center and CHat icon do not have the same margin nor animation than Notif and aEdit button

### CODE REVIEW

File: application/src/main/webapp/css/components/chatDrawer.less [hunk 1]
```diff
@@ -2,7 +2,8 @@
 @contactDefaultColor:#ffffff;
 @contactSelected:#E1E8EE;
 @subTitleColor:#a8b3c5;
-@unreadMessage:#8B0000;
+@unreadMessage:#ff5335;
+@iconDrawerColor:#a8b3c5;
```
Comment (ahamdi) [inline application/src/main/webapp/css/components/chatDrawer.less:6]: Shouldn't those be in  variables.less or couldn't we use some that already exists there ?

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 2]
```diff
@@ -304,6 +305,17 @@ export default {
         tiptip.style.display = 'none';
       }
     },
+    truncateName(text) {
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:308]: Try to use CSS for this .

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:308]: i already try to do it with text-overflow: ellipsis but it doesn't work

File: application/src/main/webapp/css/components/chatDrawer.less (hunk 1 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/css/components/chatDrawer.less]: done

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 2 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: done

**Pull request merged**